### PR TITLE
Attemt to solve NumatoGpioError when response is not lowercase 

### PR DESCRIPTION
**Pull request by lwej**
Hi, I have also encountered issue #14 and in my tests - this solution seems to work

I have not yet found a solution for unexpected line endings, but currently working on this..

### CODE REVIEW

**Comment by lwej** [comment]
I am wondering if the unexpected line ending issue might be related to port range

When calling `adc_read` on 64 port module, I get the following errors:

```
Traceback (most recent call last):
  File "/..../python3.9/site-packages/numato_gpio/__init__.py", line 227, in adc_read
    try:              return int(resp[: resp.find(self._eol)])
ValueError: invalid literal for int() with base 10: ''

Traceback (most recent call last):
  File "/..../gpio.py", line 119, in <module>
    print(dev.adc_read(3))
  File "/..../python3.9/site-packages/numato_gpio/__init__.py", line 230, in adc_read
    raise NumatoGpioError(
numato_gpio.NumatoGpioError: Query ''adc read 3'' returned unexpected result '>'. Expected 10 bit decimal integer.
```

From what I can understand, this is because `adc_read` might be the only function using port directly  ?

File: src/numato_gpio/__init__.py [hunk 1]
```diff
@@ -436,7 +441,7 @@ def _write(self, query):
 
     def _read_string(self, expected):
         string = self._read(len(expected.encode()))
-        if string != expected:
+        if string.lower() != expected:
```
Comment (clssn) [inline src/numato_gpio/__init__.py:444]: Please add a comment from which device type you expect a response in capital letters.

**Review by clssn — CHANGES_REQUESTED** [review]
I overlooked this reading the docs. I have only 32 port devices so couldn't test a real device. Thanks for your contribution!

**Comment by lwej** [comment]
> I overlooked this reading the docs. I have only 32 port devices so couldn't test a real device. Thanks for your contribution!

Thank you!

**Review by clssn — APPROVED** [review]

**Pull request merged**